src/components/VideoPlayerControls/VideoPlayerControlsPip.vue

```diff
@@ -35,12 +35,15 @@ export default {
       pip: (state) => state.pip,
       video: (state) => state.video,
     }),
+    ...mapState('Params', {
+      viewer: (state) => state.viewer
+    })
   },
   methods: {
     togglePip() {
       if (
         !this.pip &&
-        this.video.srcObject &&
+        (this.video.srcObject || this.viewer.enableDRM) &&
```

Comment:
There are a are a few instances where we check if the video has a srcObject before performing actions. With drm the video will never have a srcObject.